---
int32 result
uint8 majorVersion
uint8 minorVersion
uint8 revision
